systest/loader-benchmark/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
         source: $GOPATH/bin
         target: /gobin
         read_only: true
-    command: /gobin/dgraph zero -o 100 --my=zero1:5180 --logtostderr --bindall
+    command: dgraph zero -o 100 --my=zero1:5180 --logtostderr --bindall
```

Comment:
Can the above /gobin bind point be removed?

Reply:
fixed, thanks